tests/test_examples.py

```diff
@@ -960,6 +960,43 @@ def test_virtual_specs_override(tmp_path, request, monkeypatch):
         )
 
 
+@pytest.mark.skipif(not ON_CI, reason="Run on CI only")
+def test_condabin(tmp_path, request, monkeypatch):
+    input_path = _example_path("condabin")
+    for installer, install_dir in create_installer(input_path, tmp_path):
+        if installer.suffix == ".exe":
+            options = ["/AddToPath=1"]
```

Comment:
I think the test is failing on Windows because you're admin on GitHub runner.

Reply:
I tried my best as you can see in all the commits underneath, but there seems to be a permissions issue to run `pytest` from the non-admin account. That said, I checked on the VM and:

1. The test doesnt' pass there either
2. BUT the PATH environment var modification can be seen in the Control Panel dialog.
3. The assertion does pass in a new terminal window

I tried several ways to start a fresh process (`subprocess` flags, `os.startfile`...), and no luck.

I did check locally that the PATH is indeed modified, so it works, but it can't be tested on CI :/

Reply:
So, it looks like you essentially have to restart your shell and environment variables do not propagate into the child processes `pytest` starts. Does checking the registry work on your local system?

If so, I have the following suggestion that actually checks two things at once:

* Check if user is an admin via `ctypes.windll.shell32.IsUserAnAdmin()`
* Force the installation to use `/InstallationType=JustMe`.
* If the user is an admin, mark the test as `xfail`. This will also check whether the CVE that disables adding to path with an admin installation hasn't been re-introduced.
* Parametrize the test to check for both `classic` and `condabin` - I don't think there is a test for that yet.